Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/PerformanceCounterFromJsonGauge.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation
+{
+    using Microsoft.ApplicationInsights.DataContracts;
+
+    /// <summary>
+    /// Gauge that gives the user an aggregate of requested counters in a cache
+    /// </summary>
+    internal class PerformanceCounterFromJsonGauge : ICounterValue
```

Comment:
for this repo - can you please come up with the better names for those gauges? RawValueGauge or something